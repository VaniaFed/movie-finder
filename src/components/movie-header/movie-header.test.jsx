import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router';

import { MovieHeader } from './index';

describe('<MovieHeader />', () => {
    const tree = renderer
        .create(
            <MemoryRouter>
                <MovieHeader />
            </MemoryRouter>
        )
        .toJSON();
    it('should be rendered', () => {
        expect(tree).toMatchSnapshot();
    });
});
