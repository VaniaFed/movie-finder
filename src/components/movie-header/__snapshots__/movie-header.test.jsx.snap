// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieHeader /> should be rendered 1`] = `
<div
  className="movie-header"
>
  <h1
    className="logo"
  >
    netfixroulette
  </h1>
  <a
    href="/search"
    onClick={[Function]}
  >
    <button
      className="movie-header__search-button movie-header__search-button--reversed button"
      data-button="search-page__button-search"
      type="button"
    >
      search
    </button>
  </a>
</div>
`;
